packages/lexical-devtools/public/devtools.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+// eslint-disable-next-line no-undef
+chrome.devtools.panels.create('Lexical', 'logo192.png', 'index.html', () => {});
```

Comment:
To remove the lint warning: 
```suggestion
chrome.devtools.panels.create('Lexical', 'logo192.png', 'index.html', () => {
    return;
});
```

Reply:
> To remove the lint warning:

@thegreatercurve Ah yeah, ESLint was actually signalling `chrome` as undefined here. It seems like `chrome` is defined in Firefox, Chrome, and Edge. Maybe `let chrome = chrome` would be a way to get around the linter without disabling it? Open to suggestions here.

Reply:
Maybe similar to this: https://github.com/facebook/lexical/blob/c8b7add27d1435972a586a1c89130ba776748914/.eslintrc.js#L14